deps/FalkorDB-core-rs/README.md

```
@@ -0,0 +1,281 @@
+4. [Use from any client](#using-falkordb)
+5. [Documentation](https://docs.falkordb.com/)
+
+## Give it a try
+
+To quickly try out FalkorDB, launch an instance using docker:
+
+```
+docker run -p 6379:6379 -it --rm falkordb/falkordb:edge
+```
+
+Once loaded you can interact with FalkorDB using any of the supported [client libraries](#Client-libraries)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
Expected: #client-libraries; Actual: #Client-libraries

(MD051, link-fragments)